Error formatting if have single open bracket

This template is a guideline, not a strict requirement.
- **Version**: 2.6.0
- **Integration**: `sbt scalafmtAll` via terminal
- **Configuration**:
```
version = 2.6.0
align = more
maxColumn = 120
docstrings = ScalaDoc
align.openParenCallSite = true
align.openParenDefnSite = true
align.tokens = [
  {code = "=>", owner = "Case"},
  {code = "%", owner = "Term.ApplyInfix"},
  {code = "%%", owner = "Term.ApplyInfix"}
]
newlines.alwaysBeforeTopLevelStatements = true
rewrite.rules = [PreferCurlyFors, SortImports]
optIn.breakChainOnFirstMethodDot = false

```

## Steps

Given code like this:
```scala
class Test {

  val testCases = List(
    ("Old state", "Campaign data", "Expected state"),
    (
      1,
      2,
      3
    )
  )
}
```

When I run scalafmt like this:

```bash
sbt scalafmtAll
```

## Problem

Error is thrown:
```bash
[error] .../Test.scala:7: error: Unable to format file due to bug in scalafmt
[error]     (
[error]     ^: ...Test.scala
```

## Expectation

I would like the code to be formatted

## Workaround

I've found that by not letting single bracket to be on the line, the formatter will work. e.g.:
```scala
class Test {

  val testCases = List(
    ("Old state", "Campaign data", "Expected state"),
    (1, 2, 3)
  )
}
```

## Notes

Used `org.scalameta:sbt-scalafmt_2.12:2.4.0` plugin version. Original code, on which scalafmt fails, is valid and compiles. With 2.5.3 it works fine

--- a/scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala
+++ b/scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala
@@ -774,7 +774,7 @@ class Router(formatOps: FormatOps) {
           else style.align.openParenCallSite
         } && (!handleImplicit ||
           style.newlines.forceAfterImplicitParamListModifier)
-        val alignTuple = align && isTuple(leftOwner)
+        val alignTuple = align && isTuple(leftOwner) && !onlyConfigStyle
 
         val keepConfigStyleSplit = !sourceIgnored &&
           style.optIn.configStyleArguments && newlines != 0

--- a/scalafmt-tests/src/test/resources/default/Apply.stat
+++ b/scalafmt-tests/src/test/resources/default/Apply.stat
@@ -1374,4 +1374,13 @@ class Test {
   )
 }
 >>>
-BestFirstSearch:327 Failed to format
+class Test {
+  val testCases = List(
+      ("Old state", "Campaign data", "Expected state"),
+      (
+          1,
+          2,
+          3
+      )
+  )
+}

--- a/scalafmt-tests/src/test/resources/default/Apply.stat
+++ b/scalafmt-tests/src/test/resources/default/Apply.stat
@@ -1360,3 +1360,18 @@ object a {
       "Startup timed out. This is usually related to actor system host setting or host name resolution misconfiguration.",
       e)
 }
+<<< #2033
+align.openParenCallSite = true
+===
+class Test {
+  val testCases = List(
+    ("Old state", "Campaign data", "Expected state"),
+    (
+      1,
+      2,
+      3
+    )
+  )
+}
+>>>
+BestFirstSearch:327 Failed to format

